spec/views/recipe_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe 'The recipes index page', type: :feature do
+  it 'shows all recipes' do
+    visit '/recipes'
+    expect(page).to have_content('Recipes')
+  end
+end
```

Comment:
- [ ] Great job including tests :clap: Currently, One of your tests is failing so kindly fix that as well :+1: 
- Test Failing
![image](https://github.com/ridabensalem/recipe-app/assets/102699958/6fc390a2-0fda-4a2f-8670-a15a683508f2)

Reply:
Fixed.